app/presenters/tasks/declaration.rb

```diff
@@ -9,11 +9,7 @@ def not_applicable?
     end
 
     def can_start?
-      if crime_application.pse?
-        fulfilled?(EvidenceUpload)
-      else
-        fulfilled?(CaseDetails) && crime_application.valid?(:submission)
-      end
+      fulfilled?(Review)
```

Comment:
The review task now has a good understanding of which tasks are applicable and complete. So we can defer to that task for the declaration page.